doc/contributing/backporting-to-release-lines.md

```diff
@@ -2,133 +2,95 @@
-branch, simply replace that with the staging branch for the targeted release
-line.
+Follow these steps to backport a PR (e.g., #123) to the `vN.x` release line:
 
-### Automated
+### Automated process
 
-1. Make sure you have [`@node-core/utils`][] installed
+1. Ensure [`@node-core/utils`][] is installed.
 
-2. Run the [`git node backport`][] command
+2. Execute [`git node backport`][] command:
 
-```bash
-# Backport PR 123 to v20.x-staging
-git node backport 123 --to=20
-```
+   ```bash
+   # Example: Backport PR 123 to vN.x-staging
+   git node backport 123 --to=N
+   ```
 
-3. Jump to step 5 in the Manual section below
+3. Proceed to step 5 in the Manual section below.
 
-### Manually
+### Manual process
 
-1. Checkout the staging branch for the targeted release line.
+1. Check out the `vN.x-staging` branch.
 
-2. Make sure that the local staging branch is up to date with the remote.
+2. Update the local staging branch from the remote.
 
-3. Create a new branch off of the staging branch, as shown below.
+3. Create a new branch based on `vN.x-staging`:
 
    ```bash
-   # Assuming your fork of Node.js is checked out in $NODE_DIR,
-   # the origin remote points to your fork, and the upstream remote points
-   # to git@github.com:nodejs/node.git
-   cd $NODE_DIR
-   # If v20.x-staging is checked out `pull` should be used instead of `fetch`
    git fetch upstream v20.x-staging:v20.x-staging -f
-   # Assume we want to backport PR #10157
-   git checkout -b backport-10157-to-v20.x v20.x-staging
-   # Ensure there are no test artifacts from previous builds
-   # Note that this command deletes all files and directories
-   # not under revision control below the ./test directory.
-   # It is optional and should be used with caution.
-   git clean -xfd ./test/
+   git checkout -b backport-123-to-v20.x v20.x-staging
    ```
 
-4. After creating the branch, apply the changes to the branch. The cherry-pick
-   will likely fail due to conflicts. In that case, you will see something
-   like this:
+4. Resolve conflicts during cherry-pick:
 
    ```console
-   # Say the $SHA is 773cdc31ef
-   $ git cherry-pick $SHA # Use your commit hash
-   error: could not apply 773cdc3... <commit title>
-   hint: after resolving the conflicts, mark the corrected paths
-   hint: with 'git add <paths>' or 'git rm <paths>'
-   hint: and commit the result with 'git commit'
+   git cherry-pick <commit hash>
    ```
 
-5. Make the required changes to remove the conflicts, add the files to the index
-   using `git add`, and then commit the changes. That can be done with
-   `git cherry-pick --continue`.
+5. Resolve conflicts using `git add` and `git cherry-pick --continue`.
 
-6. Leave the commit message as is. If you think it should be modified, comment
-   in the pull request. The `Backport-PR-URL` metadata does need to be added to
-   the commit, but this will be done later.
+6. Keep the commit message unchanged or modify as needed.
```

Comment:
```suggestion
6. Leave the commit message as is. If you think it should be modified, comment
   in the pull request. The `Backport-PR-URL` metadata does need to be added to
   the commit, but this will be done later.
```